src/dcqc/main.py

```diff
@@ -171,3 +174,33 @@ def list_tests():
     writer = DictWriter(sys.stdout, fieldnames)
     writer.writeheader()
     writer.writerows(rows)
+
+
+@app.command()
+def qc_file(
+    input_file: Path = input_path_arg,
+    file_type: str = file_type_opt,
+    metadata: str = metadata_opt,
+    required_tests: List[str] = required_tests_opt,
+    skipped_tests: List[str] = skipped_tests_opt,
+):
+    """Perform quality control on a single file"""
```

Comment:
Could we add in the docstring that all external tests are excluded? and/or indicate that this is an "internal" function?

Reply:
Done!

```
❯ dcqc qc-file --help
Usage: dcqc qc-file [OPTIONS] INPUT_FILE

  Run QC tests on a single file (external tests are skipped)

Arguments:
  INPUT_FILE  Input file  [required]

Options:
  -t, --file-type TEXT       File type  [required]
  -m, --metadata TEXT        File metadata  [default: {}]
  -r, --required-tests TEXT  Required tests
  -s, --skipped-tests TEXT   Skipped tests
  --help                     Show this message and exit.
```